deploy/vagrant/create

```diff
@@ -1,41 +1,35 @@
 #!/bin/bash
 
-RED='\033[1;31m'
-PURPLE='\033[1;35m'
-NO_COLOR='\033[0m'
+set -e
 
+BOLD_RED='\033[1;31m'
+GREEN='\033[32m'
+BOLD_PURPLE='\033[1;35m'
+NO_COLOR='\033[0m'
 error() {
- echo -e "${RED}$1${NO_COLOR}"
+ echo -e "${BOLD_RED}>>> $1${NO_COLOR}"
 }
-
 info() {
- echo -e "${PURPLE}$1${NO_COLOR}"
+ echo -e "${GREEN}>>> $1${NO_COLOR}"
+}
+purple() {
```

Comment:
It's special to show master IP because originally we use purple, this is to keep the color of this important info consistent, but I think it's fine to change the color now.